secretmanager/snippets/regional_samples/create_regional_secret.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for creating a new regional secret.
+"""
+
+import argparse
+from typing import Optional
+
+from google.cloud import secretmanager_v1
+
+
+# [START secretmanager_v1_create_regional_secret]
+def create_regional_secret(
+    project_id: str, location_id: str, secret_id: str, ttl: Optional[str] = None
```

Comment:
I tried to fix it but the owl bot post-processor committed a change reverting it. PTAL.

Reply:
Eek, acknowledged.

Reply:
Actually - it is because you didn't add a comma at the end of the last item. See my suggestion, versus the change you made: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/12475/commits/db598dd9884ae952a994e914e0463c6a68e9afac#diff-8acef7a13fcc121ba85a3fee34a9fa2f6097b7d6b932a230ffb9f6d25a5010c6L30